Podfile.lock

```diff
@@ -7,4 +7,6 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   yajl: 1597676af793b78d195deb0432afdf9746c0bdcd
 
-COCOAPODS: 0.39.0
+PODFILE CHECKSUM: 4bb604ae25517aab64b12b04025ea87e2eedaf32
+
+COCOAPODS: 1.0.1
```

Comment:
This part seems unrelated to the issue at hand (but I don't see any other changes resulting from this change).